nanome_vault/Vault.py

```diff
@@ -156,25 +156,41 @@ def get_server_url(self):
                 url += ':' + str(port)
         return url
 
+
+def create_parser():
+    """Create command line parser For Vault Plugin.
+
+    rtype: argsparser: args parser
+    """
+    parser = argparse.ArgumentParser(description='Parse Arguments to set up Vault Plugin and related Services.')
+    plugin_group = parser.add_argument_group('Base Arguments')
+    vault_group = parser.add_argument_group('Vault Arguments')
+
+    # Add arguments from shared Plugin argparser, so that --help will show all possible arguments you can pass.
+    base_parser = nanome.Plugin.create_parser()
+    for action in base_parser._actions:
+        if action.dest == 'help':
+            continue
+        plugin_group._add_action(action)
+
+    # Add Vault specific arguments
+    vault_group.add_argument('--api-key', dest='api_key', help='Key for connecting to Vault Manager API', required=False)
```

Comment:
Is there a way to hide something from help? `api_key` is meant to be an internal thing for special authentication from the Vault Plugin to Vault API.